assets/js/fetch-release.js

```diff
@@ -5,7 +5,25 @@ function getLatestRelease() {
         crossDomain: true,
         dataType: 'json',
         success: function(data) { 
-            $('#wallet-version').text('Current Wallet Version: ' + data['name']); 
+            $('#wallet-version').text('Current Wallet Version: ' + data.name); 
```

Comment:
How does this handle a failure condition, like if github is unreachable?

Reply:
It just doesn't display. It looks like this if nothing loads or while loading
![image](https://user-images.githubusercontent.com/30132912/92324805-f644c100-f012-11ea-81cc-448eb780fd04.png)

The Download from Github is a little lower on the left column because the `<br>` tags still display. I was debating about putting the link and `<br>` in their own div to show both at the same time, but that also causes more shift in the layout when the link loads